CONTRIBUTING.md

```diff
@@ -48,12 +42,29 @@ Contribute to fabricjs using a fully featured online development environment tha
 
 ### Setting up a local environment
```

Comment:
Should this section be a standalone section or should it reside under the PR section?